applications/nvidia_nim/nvidia_nim_imaging/nvidia_nim.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.2
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+extensions:
+
+# For a self-hosted NIM instance, uncomment the following lines and replace the base_url
+# with the URL of your NIM instance.
+# nim:
+#   base_url: http://0.0.0.0:8000/v1/medicalimaging/nvidia/vista-3d
```

Comment:
Would uncommenting as-is work or would the `base_url` on line 28 need to be replaced?

Reply:
We ship the configuration file to use the NVIDIA-hosted Vista 3D API by default. This line is for the users who want to use a self-hosted version of the API.

Reply:
Ah I understand now, thanks!